Tests/SwiftDocCTests/Semantics/Reference/TabNavigatorTests.swift

```diff
@@ -151,13 +151,20 @@ class TabNavigatorTests: XCTestCase {
                     @Small {
                         Hey but small.
                     }
+
+                    @Snippet(path: "Snippets/Snippets/MySnippet")
                 }
             }
             """
         }
         
         XCTAssertNotNil(tabNavigator)
-        XCTAssertEqual(problems, [])
+        XCTAssertEqual(
+            problems,
+            ["23: warning – org.swift.docc.unresolvedTopicReference"]
```

Comment:
Is there a comment we could add here explaining why we expect an unresolved topic reference warning? I think I'll find this confusing when I read it without the context of this PR.